addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/discovery/FreeboxDiscoveryService.java

```diff
@@ -99,6 +100,9 @@ public void onDataFetched(ThingUID bridge, LanHostsConfig hostsConfig) {
                     if (thingUID != null) {
                         logger.trace("Adding new Freebox Network Device {} to inbox", thingUID);
                         Map<String, Object> properties = new HashMap<>(1);
+                        if (!hostConfig.getVendorName().isEmpty()) {
```

Comment:
No the vendor property I set in thing-types.xml is for the bridge thing. Here I set the vendor property on other things (network devices).

I will check again but I think getVendorName(l never returns null.